nike_project/lib/common/Button/outlined_button_customer.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+class OutLinedButtonCustom extends StatelessWidget {
```

Comment:
I think OutLinedButtonCustoms same  ElevatedButtonCustom . You may be extends Component ElevatedButtonCustom

Reply:
i don't think so ... flutter have 3 type button ... you should use reasonable